src/sentry/rules/history/preview.py

```diff
@@ -366,27 +398,36 @@ def apply_frequency_conditions(
 
 
 def get_frequency_buckets(
-    project: Project, start: datetime, end: datetime, group_id: str
+    project: Project,
+    start: datetime,
+    end: datetime,
+    group_id: str,
```

Comment:
Hmm, not sure how snuba handles type coercion here for `group_id` but I believe the column values are ints.

Reply:
whoops I'll change all of the types for this